[enterprise-4.14] add RHEL9 info for EUS upgrades (#66180)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

[id="ocp-4-14-rhcos"]
=== {op-system-first}

[id="ocp-4-14-rhcos-rhel-9-2-packages"]
==== {op-system} now uses {op-system-base} 9.2

> {op-system} now uses {op-system-base-full} 9.2 packages in {product-title} {product-version}. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates. {product-title} 4.12 is an Extended Update Support (EUS) release that will continue to use {op-system-base} 8.6 EUS packages for the entirety of its lifecycle.

Reviewer: ```suggestion
{op-system} now uses {op-system-base-full} 9.2 packages in {product-title} {product-version}. With this update, you now have the latest [operating system?] fixes, features, and enhancements, as well as the latest hardware support and driver updates. {product-title} 4.12 is an Extended Update Support (EUS) release that will continue to use {op-system-base} 8.6 EUS packages for the entirety of its lifecycle.
```
- Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references), ensure that the pronoun "this" has a clear reference. "This enables you" is unclear -- what exactly is doing the enabling here?
- Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism), avoid phrases like "this feature enables you to" and focus the sentence instead on users rather than on a product or feature and its actions.

Author: This is a good suggestion, but this blob of text for RHEL bumps across OCP versions is more or less templated, so I am going to leave it as is for consistency

[id="ocp-4-14-rhel-9-considerations"]
===== Considerations for upgrading to {product-title} with {op-system-base} 9.2

> With this release, {product-title} {product-version} uses a {op-system-base} 9.2 based {op-system} and there are some considerations you must make before upgrading:

Reviewer: ```suggestion
With this release, {product-title} {product-version} uses a {op-system-base} 9.2 based {op-system}, and there are some considerations you must make before upgrading:
```
Insert comma before coordinating conjunction to fix run-on sentence

Consider the following more extensive revision:

`Because {product-title} {product-version} now uses a {op-system-base} 9.2 based {op-system}, consider the following before upgrading:`

This revision is intended to clarify the logical relationship between the two independent clauses rather than just joining them together with `and`, and to remove "there are", which in most cases is an unnecessary phrase.

* Some component configuration options and services might have changed between {op-system-base} 8.6 and {op-system-base} 9.2, which means existing machine configuration files might no longer be valid.

> * {op-system-base} 6 base image containers are not supported on {op-system} container hosts, but are supported on {op-system-base} 8 worker nodes. For more information, see the link:https://access.redhat.com/support/policy/rhel-container-compatibility[Red Hat Container Compatibility] matrix.

Reviewer: ```suggestion
* {op-system-base} 6 base image containers are not supported on {op-system} container hosts but are supported on {op-system-base} 8 worker nodes. For more information, see the link:https://access.redhat.com/support/policy/rhel-container-compatibility[Red Hat Container Compatibility] matrix.
```
- No comma is necessary before a subordinating conjunction.
